crates/fonticulus/src/fontinfo.rs

```diff
@@ -155,3 +156,135 @@ pub fn get_selection(info: &norad::FontInfo) -> u16 {
     };
     int_list_to_num(&selection) as u16
 }
+
+/// implementation based on ufo2ft:
+/// https://github.com/googlefonts/ufo2ft/blob/main/lib/ufo2ft/util.py#l307
+pub fn calc_code_page_ranges(unicodes: &HashSet<u32>) -> Vec<u8> {
+    let mut code_page_ranges = HashSet::new();
```

Comment:
> Is there a reason to prefer HashSet to Vec, here?

You're right, this could and probably should be a vec.

> would it make sense for us to generate the two separate vecs here, instead of doing it at the call-site? 

We could even go a step further and return a tuple containing two 32bit ints which is the desired end result.

Reply:
oh totally, that's the sort of thing that rust excels at, and then we're doing way less work.